system-int-test/test/cmr/system_int_test/search/generics_test.clj

```diff
@@ -358,3 +360,91 @@
 
                    "explicit UMM JSON version through suffix"
                    (format "application/vnd.nasa.cmr.umm+json;version=%s" (generics/current-generic-version (keyword concept-type-string))))))))))
+
+;; Test that generic draft collections non json records can be ingested and searched.
+(deftest all-generic-non-json-collection-draft-search-results-test
```

Comment:
Should we have a `draft_util.clj` namespace for these tests?

Reply:
I don't think we need a draft_util.clj.  We don't have any specific functions other than the test itself.